core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -233,15 +365,16 @@ import akka.util.UUIDComparator
       awaitingFlush match {
         case Some(replyTo) =>
           log.debug("External flush request")
-          if (sortedBuffer.nonEmpty) {
-            // TODO break into batches
-            write(sortedBuffer, Vector.empty[(Serialized, TagPidSequenceNr)], tagPidSequenceNrs, awaitingFlush)
+          if (buffer.pending.nonEmpty) {
+            // TODO break into batches - check, but this is now done as buffer always batches
+            write(nextBuffer, tagPidSequenceNrs, awaitingFlush)
           } else {
             replyTo ! FlushComplete
-            context.become(idle(sortedBuffer, tagPidSequenceNrs))
+            context.become(idle(nextBuffer, tagPidSequenceNrs))
           }
         case None =>
-          flushIfRequired(sortedBuffer, tagPidSequenceNrs)
+          log.debug("write finished, checking if a new flush is required")
```

Comment:
is this logged for each write? too much? (do we have a verbose logging flag here?)

Reply:
Removed, was when debugging